paginator.js

```diff
@@ -255,9 +255,6 @@ class View {
                 const background = this.docBackground
                 this.#iframe.style.display = 'none'
 
-                this.#vertical = vertical
```

Comment:
The this.#vertical and this.#dir assignments here are the original source of vertical and dir and should not be omitted. I'm not sure if the assignments in `#beforeRender` can be omitted. Since there are no unit tests for this, I suggest we don't touch this.

Reply:
So, these variables are initialized inside `#beforeRender` also,  which is why intended to remove that. Does this library have any unit tests? (:grinning:)

Reply:
Yeah, I see it now. I think it's safe to remove it in this way. I'm afraid there is no unit test for foliate-js. And neither for readest. This might be a problem.